src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-}
+export type Figure = {
+  shape: Shape;
+  color: Color;
 
-export class Triangle {
+  getArea(): number;
+};
 
+export function roundArea(area: number): number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
and use it here, after, fix it everywhere
```suggestion
  shape: Shape = Shape.Triangle;
```